### ISSUE TITLE
dev-server going into es6 code without babel

### ISSUE THREAD

**Issue by bartolkaruza** [bdefore/redux-universal-renderer#14]
After running npm run dev on the babel 6 version ("universal-redux": "2.0.0-beta3") I'm getting the following stack trace:
Node: v5.0.0
NPM: 3.5.1

```
[1] /Users/bartolkaruza/Development/Projects//src/routes.js:1
[1] (function (exports, require, module, __filename, __dirname) { import React from 'react';
[1]                                                               ^^^^^^
[1] 
[1] SyntaxError: Unexpected token import
[1]     at exports.runInThisContext (vm.js:53:16)
[1]     at Module._compile (module.js:404:25)
[1]     at Object.Module._extensions..js (module.js:432:10)
[1]     at Module.load (module.js:356:32)
[1]     at Function.Module._load (module.js:311:12)
[1]     at Module.require (module.js:366:17)
[1]     at require (module.js:385:17)
[1]     at setupRenderer (/Users/bartolkaruza/Development/Projects//node_modules/universal-redux/lib/server.js:106:19)
[1]     at Function.setup (/Users/bartolkaruza/Development/Projects//node_modules/universal-redux/lib/server.js:283:7)
[1]     at Object.<anonymous> (/Users/bartolkaruza/Development/Projects//node_modules/universal-redux/bin/server.js:12:12)
[1] 

```

Slightly modified bin/sever.js #13 

```
#!/usr/bin/env node
// require('../server.babel'); // babel registration (runtime transpilation for node)

var path = require('path');
var renderer = require('../lib/server').default;
var config = require(path.resolve('config/universal-redux.config.js'));

// TODO: why does this script execute twice, once before requires are resolved??
if(renderer.app) {

  // method 1
  renderer.setup(config);
  renderer.start();

  // method 2
  // renderer.setup(config);
  // renderer.start();

  // method 3
  // renderer.configure(config);
  // renderer.setup();
  // renderer.start();
} else {
  console.log('Renderer not found.');
}
```

```
import React from 'react';
import { Route } from 'react-router';
import Home from './containers/Home/Home';
import Order from './containers/Order/Order';

export default () => {
  return (
      <Route>
        <Route path="/" component={Home}/> ,
        <Route path="/orders" component={Order}/>
      </Route>
  );
};
```

**Comment by bdefore** [comment]
note the double slash before src/routes ... can you gist your
universal-redux.config.js ?

**Comment by bartolkaruza** [comment]
https://gist.github.com/bartolkaruza/052f33816af3ebb62294

**Comment by bartolkaruza** [comment]
double slashes are from removed project names :)

**Comment by bartolkaruza** [comment]
I'm trying to find the issue now, locally it does not seem to pick up any babel configuration (or cli runtime) when invoking bin/server.js. bin/server.js then reaches out to react and es6 code in the project using universal-redux.
I'm trying now to use babel-node in the npm command for starting the dev-server;

```
"betterScripts": {
    "dev-server": {
      "command": "cd node_modules/universal-redux && babel-node ./bin/server.js"
    },
    "build": {
      "command": "universal-redux-build",
      "env": {
        "NODE_PATH": "./src",
        "NODE_ENV": "production"
      }
    },
    "watch-client": {
      "command": "universal-redux-watch",
      "env": {
        "NODE_PATH": "./src",
        "UV_THREADPOOL_SIZE": 100
      }
    }
  }
```

which leads to: 
ReferenceError: [BABEL] /Users/bartolkaruza/Development/Projects//node_modules/react-redux/lib/index.js: Unknown option: base.stage
I'm thinking this has to do with missing babel preset dev dependencies in the project. I'm trying to add them next. Considering that you mentioned that those are no longer necessary in the dependent projects, I think I'm missing the intended approach for es6 on the server side in the dev server setup.

**Comment by bdefore** [comment]
@bartolkaruza do you have babel@5 installed globally? if so, can you try again after an `npm uninstall -g babel`?

also, you shouldn't need to run babel-node or cd into universal-redux to run the server. try either `universal-redux-server` or model after the JWT example on `babel6` branch https://github.com/bdefore/universal-redux/blob/babel6/examples/jwt/package.json

i notice the following warning during npm install when trying it with the node version you mention, 5.0.0:

`npm WARN engine babel-plugin-add-module-exports@0.1.1: wanted: {"npm":"^3.3.9"} (current: {"node":"5.0.0","npm":"3.3.6"})`

...but it does appear to work. I notice that I need to run `npm run dev`, stop, then run again for the server to be responsive. That's an issue I'll look into today.

**Comment by bdefore** [comment]
Also, if you can push a branch of your 2.0.0-beta3 example I may be able to diagnose better.

**Comment by bdefore** [comment]
@bartolkaruza i fixed an issue with `universal-redux-server`, published an update to 2.0.0-beta4, and left you a small PR that updates to 2.0 and works on my side: https://github.com/bartolkaruza/universal-redux-simple-example/pull/2

**Comment by bartolkaruza** [comment]
Everything is working fine now, thanks!
